store/mockstore/mocktikv/executor_test.go

```diff
@@ -107,3 +107,17 @@ func (s *testExecutorSuite) TestResolvedLargeTxnLocks(c *C) {
 	_, ok := errors.Cause(pairs[0].Err).(*mocktikv.ErrLocked)
 	c.Assert(ok, IsTrue)
 }
+
+func (s *testExecutorSuite) TestIssue15662(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	tk.MustExec("use test")
+
+	tk.MustExec("create table V (id int primary key, col_int int)")
+	tk.MustExec("insert into V values (1, 8)")
+
+	tk.MustExec("create table F (id int primary key, col_int int)")
+	tk.MustExec("insert into F values (1, 8)")
+
+	tk.MustQuery(" SELECT table1.`col_int` AS field1, table1.`col_int` AS field2 FROM V AS table1 LEFT JOIN F AS table2 ON table1.`col_int` = table2.`col_int` ORDER BY field1, field2 DESC LIMIT 2 ;")
```

Comment:
```suggestion
	tk.MustQuery("SELECT table1.`col_int` AS field1, table1.`col_int` AS field2 FROM V AS table1 LEFT JOIN F AS table2 ON table1.`col_int` = table2.`col_int` ORDER BY field1, field2 DESC LIMIT 2 ;")
```

Please also check the result.

Reply:
@zz-jason fixed, thx